src/NUnitFramework/framework/Interfaces/TNode.cs

```diff
@@ -321,11 +321,12 @@ private static NodeList ApplySelection(NodeList nodeList, string xpath)
         private static readonly Regex InvalidXmlCharactersRegex = new Regex("[^\u0009\u000a\u000d\u0020-\ufffd]|([\ud800-\udbff](?![\udc00-\udfff]))|((?<![\ud800-\udbff])[\udc00-\udfff])");
         private static string EscapeInvalidXmlCharacters(string str)
         {
-            if (str == null) return null;
+            if (string.IsNullOrEmpty(str)) return str;
 
             // Based on the XML spec http://www.w3.org/TR/xml/#charsets
             // For detailed explanation of the regex see http://mnaoumov.wordpress.com/2014/06/15/escaping-invalid-xml-unicode-characters/
 
+            str = str.Replace("\\", "\\\\");
```

Comment:
It's not because backslashes are invalid, otherwise I would be adding more invalid characters...

The backslashes are being escaped so that the escaped unicode characters can be properly unescaped ([See last comment in PR #1787](https://github.com/nunit/nunit/issues/1787)). This allows users to enter sequences such as `\\x1b001` and still print `\x1b001` on the screen (without it being converted to the character).